### PR TITLE
bump Pretender version

### PR DESCRIPTION
**Pull request by givanse**
~- [ ] depends on https://github.com/pretenderjs/pretender/pull/248~
- [x] new PR https://github.com/pretenderjs/pretender/pull/255

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,11 +63,12 @@
   },
   "dependencies": {
     "abortcontroller-polyfill": "^1.1.9",
+    "broccoli-concat": "^3.7.3",
     "broccoli-funnel": "^2.0.1",
     "broccoli-merge-trees": "^3.0.0",
     "ember-cli-babel": "^6.6.0",
     "fake-xml-http-request": "^2.0.0",
-    "pretender": "^2.1.0",
+    "pretender": "^3.0.0",
```
Comment (givanse) [inline package.json]: Currently just a tag in the repo, but not released yet.

File: index.js [hunk 2]
```diff
@@ -25,6 +26,13 @@ module.exports = {
       destDir: '/pretender',
     });
 
+    pretenderTree = concat(pretenderTree, {
+      outputFile: '/pretender/pretender.js',
+      footer: 'define(\'pretender\', [], function() {' +
+              '  return self.Pretender;' +
+              '});',
+    });
```
Comment (givanse) [inline index.js]: For the full context of why I'm doing this please see the comment in the PR
> output umd
https://github.com/pretenderjs/pretender/pull/248#issuecomment-462678871

File: package.json (hunk 1 above)
Comment (givanse) [inline package.json]: > error Couldn't find any versions for "pretender" that matches "^3.0.0"

v3.0.0 is a tag, but not released yet

**Comment by givanse** [comment]
pretender 3.0.1 is out 👍

**Comment by xg-wang** [comment]
Can we merge this PR?

**Comment by rwjblue** [comment]
Seems fine, but I can't tell if I need to release as a major bump or not. The changelog in `pretender` is not terribly helpful (it says it dropped support for Node 6, yet this PR still passes on Node 6 CI).

**Comment by givanse** [comment]
This addon is consuming the already built IIFE from Pretender. So it shouldn't matter what Node version Pretender supports. Should only be relevant if you are building it yourself.

**Pull request merged**